seacrowd/sea_datasets/seaeval/seaeval.py

```diff
@@ -0,0 +1,238 @@
+            description=f"{_DATASETNAME}_{subset} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}_{subset}",
+        )
+        for subset in SUBSETS
+    ]
+
+    BUILDER_CONFIGS += [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{subset}_seacrowd_qa",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME}_{subset} SEACrowd schema",
+            schema="seacrowd_qa",
+            subset_id=f"{_DATASETNAME}_{subset}",
+        )
+        for subset in SUBSETS
+    ]
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source" and self.config.subset_id not in ["cross_logiqa", "ph_eval"]:
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "choices": datasets.Sequence(datasets.Value("string")),
+                    "answer": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "source" and self.config.subset_id == "cross_logiqa":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "context": datasets.Value("string"),
+                    "choices": datasets.Sequence(datasets.Value("string")),
+                    "answer": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "source" and self.config.subset_id == "ph_eval":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "choices": datasets.Sequence(datasets.Value("string")),
+                    "answer": datasets.Value("string"),
+                    "category": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "seacrowd_qa":
+            features = schemas.qa_features
+        else:
+            raise ValueError(f"Unexpected schema received! {self.config.schema}")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """
+        Returns SplitGenerators.
+        """
+
+        data = {key: dl_manager.download_and_extract(value) for key, value in _URLS.items()}
+
+        paths = {}
+        file = self.config.subset_id.split("_")
+        file = "_".join(file[1:3])
+        paths[self.config.subset_id] = data[file]
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
```

Comment:
Since SeaEval is used for evaluation only, let's.update the split into `TEST` instead of `TRAIN`